### ISSUE TITLE
Option to hide top or bottom portion of the card

### ISSUE THREAD

**Issue by almighty059** [pkissling/clock-weather-card#22]
Feature request to hide/show parts of the card which is basically only two parts. So if I want I could just show the top portion of the card with the clock, date, and current temp information or I could just show the bottom portion of the card with daily forecast. The default would be to show the entire card as it is. 

Besides that the cards works awesome and I haven't had any issues. I want to use it for my [custom:sidebar-crd](https://github.com/DBuit/sidebar-card) but I need to `style:` it a little bit so it fits because the side bar is only 25% of my screen and small than a "normal" column. I don't think that will be a problem though.

**Comment by guevara777** [comment]
This is very easy to do with Card-Mod. But i would also love to see the option for a yaml-config.

```
          card_mod:
            style: |
              clock-weather-card-forecast {
                display: none !important; #do not show the forcast-part of the card
              }
              clock-weather-card-today {
                display: none !important; #do not show the today-part of the card
              }
```

Maybe you have to play around with some other values also. I made a config with a custom:mod-card and card-mod and it looks like this:
![image](https://user-images.githubusercontent.com/29648553/196392450-7cb3a33f-a6a8-459a-9203-8ef02b8b9124.png)

**Comment by pkissling** [comment]
thanks for the suggestion. i just raised #30. you might want to have a look at it so see if that change suits you

**Comment by almighty059** [comment]
@guevara777 yes that works great and I tried doing as well as changing `font-size` using the element inspector but I suck at converting the information that I get from the inspector to the proper HA format in `style:`. I get soooo frustrated but I occasionally get in right on accident. Lol. I've searched for a video tutorial or something that explains how to do it but could never find anything. I get confused with shadow roots and using the `$` and `.` properly. If you know of anything that could help me learn how to it would be greatly appreciated. 

@pkissling it looks right from the way you describe it in the what you put in the new `readme` file and based off of what @guevara777 showed me but I don't know how to actually try it or review it. I was looking for a `.js` file to copy and paste over the current one but I only have a basic understanding of pull requests, commits, etc. so I'm stuck for now. Is there any easy way for you to explain how to try it or review it.? I wouldn't mind learning how to anyway. Thanks.

**Comment by pkissling** [comment]
> @guevara777 yes that works great and I tried doing as well as changing `font-size` using the element inspector but I suck at converting the information that I get from the inspector to the proper HA format in `style:`. I get soooo frustrated but I occasionally get in right on accident. Lol. I've searched for a video tutorial or something that explains how to do it but could never find anything. I get confused with shadow roots and using the `$` and `.` properly. If you know of anything that could help me learn how to it would be greatly appreciated.
> 
> @pkissling it looks right from the way you describe it in the what you put in the new `readme` file and based off of what @guevara777 showed me but I don't know how to actually try it or review it. I was looking for a `.js` file to copy and paste over the current one but I only have a basic understanding of pull requests, commits, etc. so I'm stuck for now. Is there any easy way for you to explain how to try it or review it.? I wouldn't mind learning how to anyway. Thanks.

hi @almighty059, the first step should be to get checkout the git repository and install `node` and `npm` to make eventually the command `npm run build` succeed. once successfully executed, you will find the `.js` file in the `dist` folder.

i merged the pull request. will be released with `1.0.5` in the next couple of days
